app/views/games/_index.html.erb

```diff
@@ -0,0 +1,55 @@
+<% if @games.present? %>
+
+  <div class="row">
+    <% @games.each do |g|  %>
+    <div class="col-md-4">
+
+        <div class="panel-result col-md-12 padding-zero">
```

Comment:
Do `col-md-12` and `padding-zero` have any effects here? See what happens if you remove them. You may need to give `.panel-result` a fixed height. Try `36rem`